airbyte-ci/connectors/connectors_qa/connectors_qa/checks/__init__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+from .assets import ENABLED_CHECKS as ASSETS_CHECKS
+from .metadata import ENABLED_CHECKS as METADATA_CORRECTNESS_CHECKS
+from .security import ENABLED_CHECKS as SECURITY_CHECKS
+from .packaging import ENABLED_CHECKS as PACKAGING_CHECKS
+from .documentation import ENABLED_CHECKS as DOCUMENTATION_CHECKS
+
+ENABLED_CHECKS = (
+    DOCUMENTATION_CHECKS
```

Comment:
I was about to suggest that you could have a list of lists and then flatten, but then I remembered how flattening lists in Python is expressed kek.